rclpy/rclpy/__init__.py

```diff
@@ -12,12 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import rclpy._rclpy__rmw_opensplice_cpp as _rclpy
+import os
 
+from rclpy import selector
 from rclpy.node import Node
 
+_rclpy = None
+
 
 def init(args):
+    global _rclpy
+    rclpy_rmw_env = os.getenv('RCLPY_IMPLEMENTATION')
+    if rclpy_rmw_env:
+        selector.select(rclpy_rmw_env)
+    _rclpy = selector.import_rmw_implementation()
```

Comment:
Should all of this be conditioned on whether or not `_rclpy` is already setup? What happens if the user does select first? Does `import_rmw_implementation` return a singleton after `select` is called? What happens on repeat calls to `init`?